### PR TITLE
[#11895] No last message when opening chat but it was visible on chat…

### LINKED ISSUE #11895: No last message when opening chat but it was visible on chat-preview
# Bug Report

## Problem

Sometimes it happens that message is received and it's visible on chat preview with unread indicator. But when opening this chat - message(s) are not visible for, relatively, quite a while (sometimes for 4-5 sec of top-level device, sometimes up to 10 seconds). Below are log for when delay of message appearance happened.

#### Expected behavior

Message is visible when opening chat 

#### Actual behavior

Message is not visible for some time (up to 10 sec) after opening chat

https://user-images.githubusercontent.com/8749671/111467864-9c455680-872d-11eb-9f2a-fc4d131b03c4.mov

### Reproduction (not 100% but what happened)

- Create two new account (UserA and UserB)
- UserA creates 1-1 chat with UserB, created GroupChats
- UserB open main Chats view.
- UserA sends text message in each chat
- UserB opens each chat and pays attention whether message appears without delay

### Additional Information

- Status version: release 1.12.1 
- Operating System: Android and iOS

#### Logs

Logs from two devices after issue noticed:
[iPhone_geth_and_logact](https://github.com/status-im/status-react/files/6155395/status_logs.25.zip)
[Android11_geth_and_logcat](https://github.com/status-im/status-react/files/6155396/Status-debug-logs.38.zip)

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #11895

QA: test cases when chat screen is opened, from different places

### CODE REVIEW

File: src/status_im/ui/screens/home/views.cljs [hunk 1]
```diff
@@ -199,13 +200,20 @@
...
-                   :navigation      :none
-                   :right-component [react/view {:flex-direction :row :margin-right 16}
-                                     [connectivity/connectivity-button]
-                                     [notifications-button]]}]
-   [chats-list]
-   [plus-button]
-   [tabbar/tabs-counts-subscriptions]])
+  (reagent/create-class
+   {:component-did-mount #(set! (.-navigationEventListener %) (.bindComponent (.events Navigation) % "home"))
+    :componentWillAppear #(do (println "MOINT" :home)
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/home/views.cljs]: > Debug code

**Review by Parveshdhull — APPROVED** [review]

**Comment by qoqobolo** [comment]
@flexsurfer thanks for your work!

### ISSUE 1: `Unknown` screen is displayed instead of the chat view when using the `Back` gesture when navigating from the chat view to the home screen on iOS

**Steps:**
1. Open any chat on ios device
2. Slowly make a gesture "back" half the screen, and without finishing it, stay on the chat view

https://user-images.githubusercontent.com/67952253/144080847-43c8e43c-b9a5-4d0a-945d-8cfb0a102b22.MP4

**Comment by flexsurfer** [comment]
thanks @qoqobolo , unfortunately we have to disable this gesture then

**Comment by qoqobolo** [comment]
@flexsurfer okay. Will log a feature request to implement the gesture on iOS and Android to cover this downgrade, as agreed in DM.
Testing is still in progress. Thanks for waiting!

**Comment by qoqobolo** [comment]
@flexsurfer thank you! No other issues were found.
PR can be merged.

**Pull request merged**